package.json

```diff
@@ -1,21 +1,33 @@
 {
-  "name": "sumo-ot-js",
+  "name": "sumo-opentelemetry",
```

Comment:
To be honest, I'd prefer our packages to use the `@sumologic` scope. In this way, it's clear we're the ones owning it, not some random developer. It adds it a more official look. I know that the existing `sumo-logger` doesn't follow that naming strategy but IMO it'd be good to do it from now on.

cc @sujoychatterjee @kiran-prasad

Reply:
I've changed the name to `@sumologic/opentelemetry-tracing`.